pages/docs/v3/cli/run.md

```diff
@@ -5,7 +5,7 @@ description: Capacitor CLI - cap run
 
 # Capacitor CLI - cap run
 
-This command first runs [`copy`](/docs/cli/copy), then it builds and deploys the native app to a target device of your choice.
+This command first runs [`sync`](/docs/cli/sync) (which builds the native app) and then deploys it to a target device of your choice.
```

Comment:
Good catch! it ran copy at the beginning but was changed to do a sync later on and the docs weren't updated.
But sync doesn't do a build, the build is done by run command, so that part was fine, can you change it?

```suggestion
This command first runs [`sync`](/docs/cli/sync), then it builds and deploys the native app to a target device of your choice.
```